src/driver/src/DefaultDriver.cpp

```diff
@@ -77,27 +75,40 @@ void runActor(Actor&& actor,
     }
 }
 
-genny::driver::DefaultDriver::OutcomeCode doRunLogic(
-    const genny::driver::DefaultDriver::ProgramOptions& options) {
-    if (options.shouldListActors) {
-        globalCast().streamProducersTo(std::cout);
-        return genny::driver::DefaultDriver::OutcomeCode::kSuccess;
-    }
-
+DefaultDriver::OutcomeCode doRunLogic(const DefaultDriver::ProgramOptions& options) {
     genny::metrics::Registry metrics;
-
     auto actorSetup = metrics.operation("Genny", "Setup", 0u);
     auto setupCtx = actorSetup.start();
 
+    if (options.runMode == DefaultDriver::RunMode::kListActors) {
+        globalCast().streamProducersTo(std::cout);
+        setupCtx.success();
+        return DefaultDriver::OutcomeCode::kSuccess;
+    }
+
+    if (options.workloadSource.empty()) {
+        std::cout << "Must specify a workload YAML file" << std::endl;
+        setupCtx.failure();
+        return DefaultDriver::OutcomeCode::kUserException;
+    }
+
     auto yaml = loadConfig(options.workloadSource, options.workloadSourceType);
     auto orchestrator = Orchestrator{};
 
+    if (options.runMode == DefaultDriver::RunMode::kEvaluate) {
+        std::cout << "Printing evaluated workload YAML file:" << std::endl;
+        std::cout << YAML::Dump(yaml) << std::endl;
```

Comment:
As it is this will prevent doing someting like `genny --evaluate | yamllint` since the yaml and the logs are interspersed. 

Let's either:

1. dump the log messages and the file to different ostreams (e.g. logs to stderr and yaml to stdout)
2. not do logging for `--evaluate`

Reply:
Bump on this

Reply:
I went with option 2 by removing the existing "genny version" and "confirmed logging through boost" expect for option parsing errors and `--help`.

Happy to put them back and go with option 1 if you prefer.

Reply:
No I don't think the genny version thing is still valuable - it was just a smoke-test that things were compiling with boost etc I think it's cruft at this point (although I smile every time I see the little string of emojis it prints)